src/InterventionBackend.php

```diff
@@ -30,6 +35,36 @@ class InterventionBackend implements Image_Backend, Flushable
     const CACHE_DIMENSIONS = 'DIMENSIONS_';
 
     /**
+     * How long to cache each error type
+     *
+     * @config
+     * @var array Map of error type to config.
+     * each config could be a single int (fixed cache time)
+     * or list of integers (increasing scale)
+     */
+    private static $error_cache_ttl = [
+        self::FAILED_INVALID => 0, // Invalid file type should probably never be retried
+        self::FAILED_MISSING => '5,10,20,40,80', // Missing files may be eventually available
+        self::FAILED_UNKNOWN => 300, // Unknown (edge case). Maybe system error? Needs a flush?
+    ];
+
+    /**
+     * This file is invalid as it is missing
+     */
+    const FAILED_INVALID = 'invalid';
+
+    /**
+     * This file is invalid because it is not image data, or it cannot
+     * be processed by the given backend
+     */
+    const FAILED_MISSING = 'missing';
```

Comment:
The description of `FAILED_MISSING` and `FAILED_INVALID` look like they've been switch, confused or both.